CODEOWNERS

```
@@ -1 +1 @@
-* @govuk-one-login/cri-orange-team @govuk-one-login/cri-orange-admins @govuk-one-login/cri-lime-team-leads @govuk-one-login/identity-sre # Team ownership with break-glass overrides
+* @govuk-one-login/cri-orange-team @govuk-one-login/cri-orange-admins @govuk-one-login/cri-lime-team-leads @govuk-one-login/cri-lime-team @govuk-one-login/kiwi-admins @govuk-one-login/kiwi-devs @govuk-one-login/identity-sre # Team ownership with break-glass overrides
```

After discussion with @gds-jbentham this should be:

```suggestion
* @govuk-one-login/cri-lime-team-leads @govuk-one-login/cri-orange-leads @govuk-one-login/kiwi-admins @govuk-one-login/identity-sre-leads @govuk-one-login/cri-frontend-team # Team ownership with break-glass overrides
```